docs/src/app/components/pages/components/badge.jsx

```diff
@@ -0,0 +1,96 @@
+let React = require('react');
+let { FontIcon, IconButton, Badge } = require('material-ui');
+let ComponentDoc = require('../../component-doc');
+let Code = require('badge-code');
+let CodeExample = require('../../code-example/code-example');
+const NotificationsIcon = require('svg-icons/social/notifications');
+const ShoppingCartIcon = require('svg-icons/action/shopping-cart');
+const FolderIcon = require('svg-icons/file/folder-open');
+const UploadIcon = require('svg-icons/file/cloud-upload');
+
+export default class BadgePage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.desc = 'This component generates a small badge to the top-right of it\'s child(ren)';
+
+    this.componentInfo = [
+      {
+        name: 'Props',
+        infoArray: [
+          {
+            name: 'badgeContent',
+            type: 'node',
+            header: 'required',
+            desc: 'This is the content rendered within the badge.',
+          },
+          {
+            name: 'backgroundColor',
+            type: 'string',
+            header: 'optional',
+            desc: 'Override the badge\'s background color.',
+          },
+          {
+            name: 'labelColor',
```

Comment:
It might be best to have only badgeStyle, which allows the user to set both backgroundColor and color, so they aren't confused by additional attributes like labelColor and backgroundColor.